medicines/doc-index-updater/src/create_manager/mod.rs

```diff
@@ -9,21 +8,18 @@ pub async fn create_service_worker() -> Result<String, AzureError> {
     let mut create_client = create_factory().await?;
 
     loop {
-        if let Ok(message) = get_message(&mut create_client).await {
+        let message_result: Result<CreateMessage, AzureError> = create_client.receive().await;
+        if let Ok(message) = message_result {
             tracing::info!("{:?} message receive!", message);
+            retrieve_file_from_sftp(message.document.file_path).await;
```

Comment:
This is just an empty method, right? Should it be a `TODO` comment so it matches the rest of this block?

Reply:
I think it's more like the others should be `todo!`s but for now I wanna get this merged and I'll be quickly resolving this in #425